tests/unittests/GraphOptzTest.cpp

```diff
@@ -334,12 +334,20 @@ TEST_F(GraphOptz, optimizeBatchNormAfterConvWithReshapeConst) {
   // Initialize to ensure that constant tensors are not optimized out.
   bindings_.allocate(filter)->getHandle().randomize(-1.0, 1.0, mod_.getPRNG());
   bindings_.allocate(bias)->getHandle().randomize(-1.0, 1.0, mod_.getPRNG());
-  ::glow::convertPlaceholdersToConstants(F_, bindings_, {input});
 
   EXPECT_EQ(F_->getNodes().size(), 4);
-  ::glow::optimize(F_, CompilationMode::Infer);
-  EXPECT_EQ(F_->getNodes().size(), 2);
-  EXPECT_EQ(countNodeKind(F_, Kinded::Kind::BatchNormalizationNodeKind), 0);
+  EXPECT_EQ(countNodeKind(F_, Kinded::Kind::BatchNormalizationNodeKind), 1);
+
+  optimizedF_ = F_->clone(F_->getName().str() + "_optimized");
+  ::glow::convertPlaceholdersToConstants(F_, bindings_, {input});
```

Comment:
@SplitInfinity As discussed in person, I added `convertPlaceholdersToConstants` for the original function as well. Interestingly, if I move this line before cloning, the same error occurs.